src/tree/AzureAccountTreeItemWithProjects.ts

```diff
@@ -3,49 +3,15 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { pathExists } from 'fs-extra';
-import { join } from 'path';
-import { workspace } from 'vscode';
-import { AzExtTreeItem, AzureAccountTreeItemBase, IActionContext, ISubscriptionContext } from 'vscode-azureextensionui';
-import { configFileName, enableLocalProjectView } from '../constants';
-import { getWorkspaceSetting } from '../utils/settingsUtils';
-import { LocalProjectTreeItem } from './localProject/LocalProjectTreeItem';
+import { AzureAccountTreeItemBase, ISubscriptionContext } from 'vscode-azureextensionui';
```

Comment:
Undoing the changes made here: https://github.com/microsoft/vscode-azurestaticwebapps/pull/295/files#diff-924234256676716751c864b426f5c007a88a1797571f3f1f469deb07272a70dc